### PR TITLE
APPSRE-8144 add apiToken for dt

### PR DESCRIPTION
**Pull request by fishi0x01**
Add apiToken to DT dependency

```
---
$schema: /dependencies/dynatrace-environment-1.yml

...

apiToken:
  path: path/to/token
  field: token-field
```

Follow-up of https://github.com/app-sre/qontract-schemas/pull/492

### CODE REVIEW

File: schemas/dependencies/dynatrace-environment-1.yml [hunk 1]
```diff
@@ -15,8 +15,11 @@ properties:
     type: string
   environmentUrl:
     type: string
+  apiToken:
```
Comment (fishi0x01) [inline schemas/dependencies/dynatrace-environment-1.yml:18]: @OliviaHY will this token be used for data ingestion or is it supposed to be an admin token that is only used to provision other tokens?

Comment (fishi0x01) [reply schemas/dependencies/dynatrace-environment-1.yml:18]: More context in https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/76998

**Review by maorfr — APPROVED** [review]

File: schemas/dependencies/dynatrace-environment-1.yml (hunk 1 above)
Comment (OliviaHY) [reply schemas/dependencies/dynatrace-environment-1.yml:18]: This token will be the admin token. I have been naming it bootstrap token but this is a minor detail. More details about three different tokens in https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/76998#note_7281232

**Review by OliviaHY — APPROVED** [review]

File: schemas/dependencies/dynatrace-environment-1.yml (hunk 1 above)
Comment (fishi0x01) [reply schemas/dependencies/dynatrace-environment-1.yml]: Ok aligned the naming now to `bootstrapToken` :+1:

**Pull request merged**